### PR TITLE
Updated Upload Page Inputs, Change in Upload Button When File is Selected

### PR DESCRIPTION
**Pull request by jennifertrin**
I updated the upload page video title and description inputs to have a label.
I updated the styling of the upload button and added state so that when a user selects a file, a verification animation shows up and the "Upload a Video" text changes to the file name.
I added a closeout x that goes back to the main feed.

### CODE REVIEW

File: components/closeOut.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React, { useState, useRef, useEffect } from "react";
+import { Icon, InlineIcon } from "@iconify/react";
+import timesSolid from "@iconify/icons-la/times-solid";
+import Link from "next/link";
+
+export default function closeOut() {
+
+    return (
```
Comment (MattNguyen) [inline components/closeOut.js:8]: I notice you're using 4 spaces here. Is the the convention? It's 2 spaces elsewhere.

File: pages/upload.js [hunk 2]
```diff
@@ -131,103 +147,125 @@ export default function Other() {
             </Alert>
           )}
           {state.gif && <img src={state.gif} width="200" height="200" />}
+
           <div className="w-full h-full">
+            <CloseOut></CloseOut>
```
Comment (MattNguyen) [inline pages/upload.js:152]: Can just do `<CloseOut />` if no children

Comment (jennifertrin) [reply pages/upload.js:152]: Thank you, I will update it on my end!

File: components/closeOut.js (hunk 1 above)
Comment (jennifertrin) [reply components/closeOut.js:8]: Um, I think I just made a mistake with the spacing. I will update it on my end. Thank you!

**Pull request closed without merging**